main.tf

```diff
@@ -1,13 +1,31 @@
 locals {
   enabled = module.this.enabled
+
+  is_vpc = var.vpc_id != null
+}
+
+data "aws_s3_bucket" "landing" {
+  bucket = var.s3_bucket_name
 }
 
 resource "aws_transfer_server" "default" {
   identity_provider_type = "SERVICE_MANAGED"
-  protocols              = ["SFTP"] # SFTP, FTPS, FTP
-  domain                 = "S3"     # EFS, S3
-  endpoint_type          = "PUBLIC" # VPC, PUBLIC
+  protocols              = ["SFTP"]
```

Comment:
Why not use a `var.protocols` here with the default as `["SFTP"]` ?

Reply:
This module is going to be renamed to `terraform-aws-transfer-sftp` so it will only support SFTP. We're going to create another module for FTP/S.